software/slcl.yml

```
@@ -0,0 +1,11 @@
+name: slcl
+website_url: https://gitea.privatedns.org/xavi/slcl
+description: Simple and lightweight web cloud storage.
+licenses:
+  - AGPL-3.0
+platforms:
+  - C
+tags:
+  - File Transfer - Object Storage & File Servers
+  - File Transfer & Synchronization
```

```suggestion
  - File Transfer - Web-based File Managers
```

I think this is more appropriate

---

But this also seems to be a File Servers, not just a web-interface. So at least `File Transfer - Object Storage & File Servers` should still be valid, or not?

---

`slcl` is assumed to be accessed from a web interface, be it a web browser, cURL, whatever. I am open to suggestions - which one do you prefer?